tests/integration/async_push_test_client.py

```diff
@@ -272,17 +273,27 @@ async def get_broadcast(self, timeout=1):  # pragma: no cover
             log.error(f"Error: {ex}")
             return None
 
-    async def ping(self):
-        """Test websocket ping."""
+    async def moz_ping(self):
+        """Send a very small message and await response."""
         if not self.ws:
             raise Exception("WebSocket client not available as expected.")
 
         log.debug("Send: {}")
-        await self.ws.ping("{}")
+        await self.ws.send("{}")
         result = await self.ws.recv()
         log.debug(f"Recv: {result}")
         return result
 
+    async def ping(self):
```

Comment:
Not really sure that this is necessary, since it's testing an aspect of the Websocket protocol, not our implementation of Web Push but 🤷

Reply:
Yeah, you are right, don't necessarily need it. I can remove